### ISSUE TITLE
All results boxed audio a11y

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse-frontend#662]
## Description

From @sarayourfriend 

Audio results need to be composite items with the play/pause button removed from the default tab order. Currently tabbing through each result is fine until you hit an audio result which will tab from the result link into the play/pause button before moving on to the next result. The tab order should just be each result. This happens both with and without VoiceOver activated.

With VoiceOver, the audio results are not announcing their title. I think this would be fixed with the composite items as I believe they require `aria-label` which would announce the title.

Audio links are also missing an `aria-label` so all audio links just read "link, Audio player" in VoiceOver.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

**Comment by zackkrida** [comment]
@WordPress/openverse-frontend I'm not sure what the best approach is here—I feel like I'm over thinking it or missing something obvious. I think the _behavior_ we want is:

1. Can only focus the outermost node of the boxed audio, not internally
	- Is the best way to do this to remove the tabindex on play-pause when the layout is `box`?
2. Pressing space or clicking the play button directly will toggle the audio playback
3. Clicking anywhere except the play button or pressing enter on the audiobox will navigate to the single result

**Comment by sarayourfriend** [comment]
I think that sounds right Zack.

> Is the best way to do this to remove the tabindex on play-pause when the layout is box?

Setting the tabindex to `-1` specifically would do it I think, yes.

> Pressing space or clicking the play button directly will toggle the audio playback

The only thing with this is that we'll have to indicate somehow to both screen reader users and folks who only rely on keyboard navigation otherwise that they can play the audio using the space bar. Under most circumstances the spacebar, when focused on a link, will scroll the page. Screen readers are probably the easier case here as we could incorporate it in the `aria-label` of the link (which is currently missing, I've updated the issue description to include that as a requirement) using a format like `` `${audio.title} - Press spacebar to play and pause the audio preview` ``.

For users who are only using keyboard navigation without a screen reader, we'll need to figure out another way to inform them about this behavior. Maybe we can show a popover above the audio result that indicates that the spacebar can be used the play/pause the audio preview, but only when we've detected that someone is using the keyboard to navigate. We'll also need to hide this information from screen reader users to avoid duplicating the information. In this case we'll need to set the `autoFocusOnShow` prop to `false` on the popover.
